apps/api/src/app/events/usecases/send-message/send-message.usecase.ts

```diff
@@ -215,4 +222,33 @@ export class SendMessage {
 
     return !channels.find((channel) => channel === job.type);
   }
+
+  private sumFilters(
+    summary: {
+      stepFilters: string[];
+      failedFilters: string[];
+      passedFilters: string[];
+    },
+    condition: ICondition
+  ) {
+    let type: string = condition.filter.toLowerCase();
```

Comment:
Is there any case that the filter will be undefined? Maybe better to add a 
```suggestion
    let type: string = condition.filter?.toLowerCase();
```

Reply:
According to the type there should always be a string...

Reply:
Yes seen it, but is it possible that there will be some data inside the DB that will be set to null or undefined?